src/codeu/chat/common/SentimentScore.java

```diff
@@ -0,0 +1,117 @@
+package codeu.chat.common;
+
+import codeu.chat.util.Serializer;
+import codeu.chat.util.Serializers;
+import com.google.cloud.language.spi.v1beta2.LanguageServiceClient;
+
+import com.google.cloud.language.v1beta2.AnalyzeSentimentResponse;
+import com.google.cloud.language.v1beta2.Document;
+import com.google.cloud.language.v1beta2.Document.Type;
+import com.google.cloud.language.v1beta2.Sentiment;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+/**
+ * Created by rsharif on 5/3/17.
+ */
+public class SentimentScore {
+
+  private int score;              // the average score for this object
+  private int numScores;          // this will mark how many messages have been included in the calculation of the current score
```

Comment:
There's a preference for self-documenting variable names here at Google.  Private variable names are especially rarely commented.  I think score is pretty well self-defined, maybe numScores can be extended a bit to capture meaning without having to comment.